sound/soc/intel/boards/skl_hda_dsp_common.c

```diff
@@ -132,6 +137,14 @@ struct snd_soc_dai_link skl_hda_be_dai_links[HDA_DSP_MAX_BE_DAI_LINKS] = {
 		.no_pcm = 1,
 		SND_SOC_DAILINK_REG(dmic16k, dmic_codec, platform),
 	},
+	{
+		.name = NULL, /* initialized in driver probe function */
+		.id = 8,
+		.dpcm_playback = 1,
+		.dpcm_capture = 1,
```

Comment:
I forgot if those flags still exist or have been removed.

Reply:
not yet removed. still doing some cleanups
https://patchwork.kernel.org/project/alsa-devel/list/?series=874792